client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -195,14 +202,22 @@ export const AuthProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
   };
 
   const login = async (username: string, password: string) => {
-    setSkipRequest(skipNoStoreRequests);
     const { token, error } = await mutateAsync({ username, password });
     setHeader('Authorization', `Bearer ${token}`);
     const store = await getStore(token);
+    const permissions = await getPermissions({
+      storeId: store?.id || '',
+      token,
+    });
+    setSkipRequest(skipNoStoreRequests);
```

Comment:
the permissions query is an auth query and would be skipped if this wasn't added after the permissions call..